charts/geoserver/templates/ingress.yaml

```diff
@@ -0,0 +1,39 @@
+{{- if .Values.ingress.enabled -}}
+{{- $fullName := include "geoserver.fullname" . -}}
+{{- $svcPort := .Values.service.port -}}
+{{- if semverCompare ">=1.14-0" .Capabilities.KubeVersion.GitVersion -}}
```

Comment:
Check, this is not correct anymore. in v1.21 this changes to be networking.k8s.io/v1

Reply:
changed